bluebutton-server-app/src/test/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/PartDEventTransformerTest.java

```
@@ -154,6 +156,15 @@ static void assertMatches(PartDEvent claim, ExplanationOfBenefit eob) throws FHI
 		if (claim.getSubmissionClarificationCode().isPresent())
 			TransformerTestUtils.assertInfoWithCodeEquals(CcwCodebookVariable.SUBMSN_CLR_CD,
 					CcwCodebookVariable.SUBMSN_CLR_CD, claim.getSubmissionClarificationCode(), eob);
+
+		try {
+
+			TransformerTestUtils.assertFDADrugCodeDisplayEquals(claim.getNationalDrugCode());
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

You should not swallow/ignore exceptions like this.